src/features/apiDelegation/components/OverviewPageContent/OverviewPageContent.tsx

```diff
@@ -178,82 +178,81 @@ export const OverviewPageContent = ({
   };
 
   return (
-    <div className={common.pageContent}>
-      <div className={classes.overviewActionBarContainer}>
-        {!isSm && <h2 className={classes.pageContentText}>{overviewText}</h2>}
-        {layout === LayoutState.Offered && (
-          <div className={classes.delegateNewButton}>
-            <Button
-              variant={ButtonVariant.Outline}
-              onClick={goToStartDelegation}
-              icon={<Add />}
-              fullWidth={isSm}
-            >
-              {t('api_delegation.delegate_new_org')}
-            </Button>
-          </div>
-        )}
-        <Panel
-          title={t('api_delegation.card_title')}
-          forceMobileLayout={isSm}
-          showIcon={!isSm}
-        >
-          {t('api_delegation.api_panel_content')}{' '}
-          <a
-            className={classes.link}
-            href='https://samarbeid.digdir.no/maskinporten/maskinporten/25'
-            target='_blank'
-            rel='noreferrer'
+    <div className={classes.overviewActionBarContainer}>
+      {!isSm && <h2 className={classes.pageContentText}>{overviewText}</h2>}
+      {layout === LayoutState.Offered && (
+        <div className={classes.delegateNewButton}>
+          <Button
+            variant={ButtonVariant.Outline}
+            onClick={goToStartDelegation}
+            icon={<Add />}
+            fullWidth={isSm}
+            size='medium'
```

Comment:
Run yarn lint and remove warnings like the one with ButtonSize not being used anymore after switching to size string

Reply:
Tried running it for about half an hour but it didn't seem to do anything 😅
Removed the leftover ButtonSizes though

Reply:
Hmm, it is processing at that point though. I think that has to do with the image using too long time to process yarn lint command, on my mac it takes around 7 seconds